removefiles/activity.go

```diff
@@ -0,0 +1,51 @@
+package removefiles
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+)
+
+const ActivityName = "remove-files-activity"
+
+type Activity struct {
+	logger logr.Logger
+}
+
+func NewActivity(logger logr.Logger) *Activity {
+	return &Activity{logger: logger}
+}
```

Comment:
Instead of passing the logger to the activity we started to get it from the context with https://pkg.go.dev/go.artefactual.dev/tools@v0.8.0/temporal#GetLogger (activities) and https://pkg.go.dev/go.temporal.io/sdk/workflow#GetLogger (workflows).

Reply:
@jraddaoui to you have any examples of https://pkg.go.dev/go.artefactual.dev/tools@v0.8.0/temporal#GetLogger being used in the activity context? I can't find any.

Reply:
A similar pattern is used here: https://github.com/artefactual-labs/enduro/blob/85c48081036e0d1bd2f7e93086bf9d1599ae8ceb/internal/workflow/activities/poll_ingest.go#L29. I could not find any usage of our temporal package in public repos, but I'll share another example in Slack ;)